pyvista/plotting/plotting.py

```diff
@@ -2558,8 +2574,9 @@ def close(self):
             renderer.close()
 
         # Grab screenshots of last render
-        self.last_image = self.screenshot(None, return_img=True)
-        self.last_image_depth = self.get_image_depth()
+        if self._store_image:
+            self.last_image = self.screenshot(None, return_img=True)
+            self.last_image_depth = self.get_image_depth()
```

Comment:
hmmm... I'm worried this might have some unintended repercussions. I agree with the change but I'm thinking back to when I *hacked* this together as there were all kinds of weird edge cases where it was just safest to always storge a screenshot on close. 

But having this `store_image` property will be a quick workaround for anyone having issues and its far more intentional than forcing a screenshot on close

Reply:
Sharp eyes for seeing this.

I merged the branch from #730 into this branch as the unit tests were taking forever with all the screenshot saving.

Reply:
agreed